src/main/webapp/i18n/de/conversation.json

```
@@ -8,6 +8,13 @@
             "mainParagraph": "Gib im Chat keine vertraulichen Informationen wie Passwörter, Kreditkartennummern oder persönliche Identifikationsnummern weiter. Diese Informationen sind sensibel und könnten für Identitätsdiebstahl oder Betrug verwendet werden, wenn sie in die falschen Hände geraten.",
             "lastParagraph": "Bewahre deine persönlichen Daten sicher auf, indem du sie für dich behältst und sie nicht online weitergibst."
         },
+        "conversation": {
+            "codeOfConduct": {
+                "topBar": {
+                    "button": "Verhaltens-Kodex"
```

```suggestion
                    "button": "Verhaltenskodex"
```
I don't see a reason for the hyphen here

---

Agree